gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystem.java

```diff
@@ -593,8 +593,8 @@ public FSDataOutputStream create(
                           .build(),
                       statistics),
                   statistics);
-          globalStorageStatistics.filesCreated();
-          instrumentation.fileCreated();
```

Comment:
why are we removing this?

Reply:
These functions hamper the usage of incrementStatistics() function. I think we should have a single function where we increment both FS based and global counter simultaneously. Also improves code readability